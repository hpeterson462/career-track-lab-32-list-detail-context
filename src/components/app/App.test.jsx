import React from 'react';
import { render } from '@testing-library/react';
import CharacterDetail from '../../components/app/characters/CharacterItem';

describe('CharacterDetail component', () => {
  it('renders CharacterDetail after loading...', () => {
    const { asFragment } = render(<CharacterDetail />);
    expect(asFragment()).toMatchSnapshot();
  });
});

