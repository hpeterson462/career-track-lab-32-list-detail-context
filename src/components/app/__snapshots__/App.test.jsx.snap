// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetail component renders CharacterDetail after loading... 1`] = `
<DocumentFragment>
  <figure>
    <img />
    <figcaption />
  </figure>
</DocumentFragment>
`;
